crosswalks/Citation_File_Format_1.2.0.README.md

```diff
@@ -0,0 +1,183 @@
+# Crosswalk for Citation File Format 1.2.0
+
+## Format information
+
+Citation File Format (CFF) is a YAML format for providing citation metdaata for software (and experimentally for datasets).
+
+- Website: <https://citation-file-format.github.io/>
+- Specifications: <https://github.com/citation-file-format/citation-file-format/blob/main/schema-guide.md>
+- Schema: <https://github.com/citation-file-format/citation-file-format/blob/main/schema.json> (JSON Schema)
+
+## Crosswalk specifics
+
+The following sections provide details for directional mapping 
+in cases where mapping is not bidirectionally unambiguous.
+
+### `downloadUrl` <=> `repository-artifact`
+
+- CodeMeta `downloadUrl`: "If the file can be downloaded, URL to download the binary."
+- CFF `repository-artifact`: "The URL of the work in a build artifact/binary repository."
+
+While the scope of `repository-artifact` is documented to be somewhat narrower than `downloadUrl`, 
+its main use case is discovery (e.g. for reuse and reproducibility),
+which is arguably the same as for `downloadUrl`.
+Therefore the two can be assumed to signify the same.
+
+Future versions of CFF may document this more clearly (https://github.com/citation-file-format/citation-file-format/issues/326).
+
+### `softwareRequirements`/`softwareSuggestions` <=> `references`
+
+- CodeMeta `softwareRequirements`: "Required software dependencies"
+- CodeMeta `softwareSuggestions`: "Optional dependencies , e.g. for optional features, code development, etc"
+- CFF `references`: "Reference(s) to other creative works."
+
+#### CodeMeta to CFF
+
+The scope of `references` could be documented more clearly in CFF (https://github.com/citation-file-format/citation-file-format/issues/327).
+CFF's `references` is meant to work like a references list for a paper, 
+e.g., should include prior work that the work builds on.
+Required dependencies are interpreted to clearly constitute prior work that software builds on.
+Therefore, everything that is in `softwareRequirements` should go into `references`.
+
+Optional dependencies and dependencies needed for code development could be assumed 
+to build on the software (i.e., require prior existence of the software).
+Therefore, the software would constitute a `reference` *from* the `softwareSuggestion`,
+but not the other way round.
+I.e., `softwareSuggestions` should *not* be included in `references`.
+
+#### CFF to CodeMeta
+
+Not all CFF `references` can be assumed to be `softwareRequirements`,
+as `references` can contain arbitrary other work that the software builds on.
+The same is even more clearly true for `softwareSuggestions`.
```

Comment:
```suggestion
The same is true for `softwareSuggestions`.
```

Reply:
Fixed in b0e3c9371175509c867c06090c85e9f6576c6ed5.